### ISSUE TITLE
Widgets

### ISSUE THREAD

**Issue by HellxDo** [TeamNewPipe/NewPipe#4192]
#### Describe the feature you want

Widgets for the home screen, a play and pause button, maybe forward 10 seconds and normal replay and forward buttons, a music player that works for background
#### How will you/everyone benefit from this feature?

I usually listen to Lofi, and my notification bar is always full, so instead of having to scroll all the way down to the new pipe notification, just put the widget on home screen and control it from there!

**Comment by Stypox** [comment]
Does the new Unified UI partially solve your problem? Check out [this apk](https://github.com/TeamNewPipe/NewPipe/files/5168628/app-debug.zip). Thank you for the descriptive report ;-)

**Comment by HellxDo** [comment]
> Does the new Unified UI partially solve your problem? Check out [this apk](https://github.com/TeamNewPipe/NewPipe/files/5168628/app-debug.zip). Thank you for the descriptive report ;-)

How do I use it exactly?

**Comment by Stypox** [comment]
@HellxDo download the apk by clicking [here](https://github.com/TeamNewPipe/NewPipe/files/5168628/app-debug.zip), open the zipped folder and open the apk. After it is installed you should find a new side installed NewPipe Debug app. That version contains the latest changes, including a new Youtube-like UI which you may like

**Comment by HellxDo** [comment]
> @HellxDo download the apk by clicking [here](https://github.com/TeamNewPipe/NewPipe/files/5168628/app-debug.zip), open the zipped folder and open the apk. After it is installed you should find a new side installed NewPipe Debug app. That version contains the latest changes, including a new Youtube-like UI which you may like

Thank you! I'll try it out

**Comment by byronmansfield** [comment]
I also would like to see this feature. @HellxDo did this suggested apk include widgets?

**Comment by Stypox** [comment]
No, it does not. It was just a preview of the features of 0.20.0

**Comment by skyGtm** [comment]
@HellxDo
@byronmansfield  
I found this app https://play.google.com/store/apps/details?id=com.widgets.music
which will do some of the tasks you mentioned
1. Play/Pause
2. Next/Previous
3. Playlist support (Depends on Widget you selected)

Hope it helps until the feature is implemented within NewPipe.
